Buildings/ThermalZones/EnergyPlus/Validation/SurfaceComparison/package.mo

```diff
@@ -0,0 +1,15 @@
+within Buildings.ThermalZones.EnergyPlus.Validation;
+package SurfaceComparison "Collection of models used to validate zone surface coupling against EnergyPlus"
+  annotation (Documentation(info="<html>
+<p>
+Package containing models used to compare ZoneSurface results against equivalent EnergyPlus surfaces
```

Comment:
Always use the full class name with `<a href=\"modelica://Buildings....`
Otherwise, if the class gets renamed, this string becomes out of date and we have no means to detected it (hyperlinks will automatically be updated by the rename scripts, and when generating html, Dymola reports invalid hyperlinks).

Reply:
Updated